### DISCUSSION TITLE
Markdown Export

### DISCUSSION THREAD

**Discussion by OtherToday** [GitbookIO/community#216] [Feature Requests]
### What's your feature/idea?

Alongside pdf export can we have markdown export. 

### What's your desired outcome?

It is great that we can upload a word doc, it would be be great to have similar download ability. The use case is for templates. I give students gitbook templates, I would like them to be able to downloads them to work off line either in an offline markdown editor or in word. At the moment i have to make two versions, a live gitbook and a word version for download that they can upload themselves. 

Another use-case is to be able work on pages offline (for example on a train).

The github integration doesn't allow this as the sync is one way and the images get hosted by github.  

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by thebiglabasky** [comment]
Hi,
Hervé from the Product team at GitBook here. 
Note that as a workaround, you can enable Git sync which will push your GitBook content as Markdown to a git repository. 
It also has the benefit of remaining up-to-date with changes made in GitBook over time and allows to work offline as long as you work using a traditional git-based workflow. 
That means it's unlikely we'd prioritize as hoc markdown exports  in the immediate future

**Reply by istvs** [reply]
Hi Hervé — piggybacking on this existing Markdown thread to ask about support for linebreaks? Shift-return is something I find myself using often and in the app it looks like it's being honored while in the published website it is not. 

Seems so silly to have to ask for <br> support but here I am! Many thanks for any guidance, searching on linebreaks/line breaks in the help docs returned no results.

**Reply by thebiglabasky** [reply]
We have an issue filed for that in the backlog that I hope we can get to soon enough. Will keep this thread posted!
